projects/aca-playwright-shared/src/page-objects/components/dataTable/data-table.component.ts

```diff
@@ -78,6 +78,13 @@ export class DataTableComponent extends BaseComponent {
   getRowByColumnTitleAndItsCellValue = (columnTitle: string, cellValue: string | number): Locator =>
     this.page.locator(`//div[contains(@title, '${columnTitle}')]//span[contains(text(), '${cellValue}')]/ancestor::adf-datatable-row`);
 
+  /**
+   * Method used in cases where we want to retrieve a row from the datatable based on its numerical order within the array.
+   * 
+   * @returns reference to cell element which contains text.
+   */
+  getNthRow = (orderNum: number): Locator => this.getChild(`adf-datatable-row`).nth(orderNum);
```

Comment:
You can use this.this.getRowLocator instead of this.getChild(`adf-datatable-row`) to don't duplicate the same selector